webapp/dojo_news/src/App.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div>
+    <h1>News</h1>
+    <div
+        v-for="item in sortedNewsListItems"
+        :key="item.title">
```

Comment:
This is weird, I'm expecting to see runtime errors because of dulicate keys. I don't know, why they are not being shown in the JavaScript console. When I open my VueJS devtools I can see that VueJS has troubles to display those components that have the same `:key` (one is always missing in the virtual DOM layout).

You should make sure that `:key` is always unique.

Reply:
Thanks!
Yes. It worked and we're aware that it's not really great design. We wanted to wait and see how the exercises develop to not optimize prematurely (adding an ID is of course an easy and almost always proper way to go).